acceptance_tests/features/steps/sample_loading.py

```diff
@@ -42,6 +48,19 @@ def load_sample_file_step(context, sample_file_name):
     context.emitted_cases = get_emitted_cases_and_check_against_sample(sample_rows)
 
 
+@step('sample file "{sample_file_name}" with sensitive column {sensitive_column} is loaded successfully')
+def load_sample_file_step_for_sensitive_data(context, sample_file_name, sensitive_column):
+    sample_file_path = Config.RESOURCE_FILE_PATH.joinpath('sample_files', sample_file_name)
+    sample_rows, sample_validation_rules = get_sample_rows_and_validation_rules(sample_file_path, sensitive_column)
+
+    context.survey_id = add_survey(sample_validation_rules)
+    context.collex_id = add_collex(context.survey_id)
+
+    upload_sample_file(context.collex_id, sample_file_path)
+
+    context.emitted_cases = get_emitted_cases_and_check_against_sample(sample_rows, sensitive_column)
```

Comment:
You're passing the single sensitive column in here where it expects a container, so the `in` check is comparing to a single string and could do unexpected things if one sample column name was a substring of another.

```suggestion
    context.emitted_cases = get_emitted_cases_and_check_against_sample(sample_rows, {sensitive_column})
```